x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -107,7 +107,7 @@ interface QueryProps {
 }
 
 const Query = ({ query, 'data-test-subj': dataTestSubj = 'query' }: QueryProps) => {
-  const styles = useQueryStyles();
+  const styles = queryStyles;
```

Comment:
nit: This constant is redundant. If you just wanna a shorter name it's better to use `as` like `queryStyles` as `styles` in import statement.

Reply:
@maximpn We can't use `import { queryStyles as styles } from ...` here because in this file we're importing styles for other components as well. So we have to keep it as `import { queryStyles } from ...`. But inside the `Query` component I'd like to shorten the name.